### ISSUE TITLE
Wrong error-message DiContainer.getInstance with name?

### ISSUE THREAD

**Issue by tieskedh** [edvin/tornadofx#1152]
The current implementation of DiContainer.getInstance with name is:

> **_Injector is not configured, so bean of type $type with name $name can not be resolved._**

Ironically, this exception will only be thrown if the DiContainer is configured.
The real reason for this exception is that the DiContainer doesn't support searching instances by name.

When we change this, this can be a breaking change. Can we just change this?

If we can change this, we the new message could be:

> **_The configured injector does not support searching instances by name, so bean of type $type with name $name can not be resolved._**

We can also be more instructional, like:
> **_The DiContainer.getInstance(type, name) is not implemented. Therefor the bean of type $type with name $name can not be resolved._**

Or is it better to go al the way and throw an NotImplementedError with either one of the messages?

**Comment by edvin** [comment]
Good catch! I think NotImplementedError with the last message might be a good idea.

**Comment by ctadlock** [comment]
Did this fix get incorporated in?

**Comment by tieskedh** [comment]
@ctadlock, sorry, I didn't take action on it.
Feel free to create the PR.
